engine/lib/phx/src/window/winit_window.rs

```diff
@@ -177,9 +177,14 @@ impl WinitWindow {
 
         window_attributes = window_attributes.with_title(window.title.as_str());
 
+        #[cfg(cgl_backend)]
+        let transparency = true;
+        #[cfg(not(cgl_backend))]
+        let transparency = false;
+
         let template = ConfigTemplateBuilder::new()
             .with_alpha_size(8)
-            .with_transparency(cfg!(cgl_backend));
+            .with_transparency(transparency);
```

Comment:
Maybe we can just delete this line or set `with_transparency` to false? That way we don't need to depend on us being on the CGL (macOS) backend

Reply:
Removed